templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -462,13 +462,24 @@
               {
                 "Variable": "$.wasReconciled",
                 "BooleanEquals": true,
-                "Next": "Do nothing"
+                "Next": "Publish reconciliation update to dr2-notifications"
               }
             ],
             "Default": "Throw Reconciler job error"
           },
-          "Do nothing": {
-            "Type": "Pass",
+          "Publish reconciliation update to dr2-notifications": {
+            "Type": "Task",
+            "Resource": "arn:aws:states:::sns:publish",
+            "Parameters": {
+              "Message": {
+                "reconciliationUpdate": "Asset was reconciled",
+                "assetId.$": "$.AssetInfo.assetId",
+                "messageId": "$.messageId",
+                "parentMessageId": "$.parentMessageId",
+                "executionId": "$.executionId"
+              },
+              "TopicArn": "arn:aws:sns:eu-west-2:${account_id}:${ingest_dr2_notifications_name}"
+            },
```

Comment:
How difficult would it be to wrap this in a MAP state and send 10 at a time using the [PublishBatch](https://docs.aws.amazon.com/sns/latest/api/API_PublishBatch.html) endpoint?

Reply:
But isn't this already in a MAP? What would we be mapping over in this case?

Reply:
Ahh, that makes it difficult. Let's drop this requirement until we need it then. It's $0.50/million requests which would be a nice cost saving when we do migrations.

Follow up but happy for this to be a backlog ticket - if the reconciler received a list of asset ids, would it cope and output a value per asset? I guess the other option is to map(batch=1) within a map(batch=10).

Reply:
It can't handle a list of asset Ids in its current state but I guess it could, although, it would be a little messy since you'd have to wrap the current Lambda code in a map. Might be worth creating a ticket so that we could discuss it.